_data/master/navbars/usage.yml

```diff
@@ -17,8 +17,14 @@ toc:
       path: /usage/calicoctl/configure/etcd
     - title: Kubernetes API datastore options
       path: /usage/calicoctl/configure/kdd
-- title: Securing Calico's Prometheus endpoints 
-  path: /usage/policy/secure-metrics
+- title: Securing Calico communications
```

Comment:
I think we can come up with a better title than this - as it is its a bit vague, and I don't immediately know if it means securing communications between Calico components or workloads deployed on top of Calico.

Maybe "Installing Calico securely" or "Securing a Calico installation" or "Configuring TLS and securing Calico ports" or something like that?

Reply:
I can see what you mean, although I am not crazy about any of the suggestions thus far. I am going to change it to `Securing Calico component communications` for now. See if that reads any better.